apps/bluetooth/README.md

```diff
@@ -0,0 +1,40 @@
+# Bluetooth service
+
+  This is the app for run bluetooth service which will handle most bluetooth events in background.
+
+## App URL
+
+* scheme and host:
+
+  * `yoda-app://bluetooth`
+
+* paths and queries:
+
+  * `open`: Open bluetooth.
```

Comment:
pathname should be started with `/`. like `/open` would be better ?

Reply:
yes, modified.